pyathena/sqlalchemy_athena.py

```diff
@@ -320,7 +410,7 @@ class AthenaDialect(DefaultDialect):
     _pattern_data_catlog_exception = re.compile(
         r"(((Database|Namespace)\ (?P<schema>.+))|(Table\ (?P<table>.+)))\ not\ found\."
     )
-    _pattern_column_type = re.compile(r"^([a-zA-Z]+)($|\(.+\)$)")
+    _pattern_column_type = re.compile(r"^([a-zA-Z]+)($|[(<].+[)>]$)")
```

Comment:
The API returns the same syntax as DDL statements for composite types, e.g. `array<int>` instead of `array(int)` when using SQL & the information schema.